code/api/src/migrations/7-user-add-address.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return Promise.all([
+      queryInterface.addColumn('users', 'address_line1', {
+        type: Sequelize.STRING
+      }),
+      queryInterface.addColumn('users', 'address_line2', {
+        allowNull: true,
+        type: Sequelize.STRING
+      }),
+      queryInterface.addColumn('users', 'city', {
+        type: Sequelize.STRING
+      }),
+      queryInterface.addColumn('users', 'state', {
+        type: Sequelize.STRING
+      }),
+      queryInterface.addColumn('users', 'zipcode', {
+        type: Sequelize.INTEGER
+      })
+    ]);
+  },
+
+  down: (queryInterface, Sequelize) => {
+    return Promise.all([
+      queryInterface.removeColumn('users', 'address_line1'),
+      queryInterface.removeColumn('users', 'address_line2'),
+      queryInterface.removeColumn('users', 'city'),
+      queryInterface.removeColumn('users', 'state'),
+      queryInterface.removeColumn('users', 'zipcode')
+    ]);
+  }
+};
```

Comment:
I'm not sure if it convention or necessary, but I appreciate making each input area a column for organization sake. This will help us when we create our form. We will also know which information you will need.

Reply:
Yay! I had a choice to create each column as a separate migration file, however, since all 5 columns here pertain to shipping address, I figured this was okay. However, in the future, if we do need to migrate/rollback individual columns more readily, this can be fixed and updated